skimage/measure/_ccomp.pyx

```diff
@@ -389,14 +386,15 @@ def label(input, neighbors=None, background=None, return_num=False,
         **Deprecated, use ``connectivity`` instead.**
     background : int, optional
         Consider all pixels with this value as background pixels, and label
-        them as -1. (Note: background pixels will be labeled as 0 starting with
-        version 0.12).
+        them as 0. If ``None``, 0-valued pixels are considered as background
```

Comment:
In my opinion, the only reason to use None is for parameters that are
automatically determined from the input or deprecations.